repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/files/FileHelper.java

```diff
@@ -0,0 +1,186 @@
+package fr.inria.spirals.repairnator.process.files;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
+import fr.inria.spirals.repairnator.Utils;
+import fr.inria.spirals.repairnator.process.step.AbstractStep;
+
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by Tellström on 05/07/18.
+ */
+
+public class FileHelper {
```

Comment:
It looks like this class could be a static one: the only "state" it holds is about the logger, which can be static. So I think you can refactor all methods to be static then you won't need to create an instance of the file helper which seems currently really useless. WDYT?

Reply:
I agree, it is only once called from ProjectInspector which could have been a concern (unsure of just what you want in that class), so it doesn't change how we use it either really.